frontend/app/src/composables/history/events/events-query-status.ts

```diff
@@ -3,8 +3,8 @@ import { type HistoryEventsQueryData } from '@/types/websocket-messages';
 
```

Comment:
I must be forgot to do this. Can you move this file to inside `/composables/history/events/query-status/`